stdlib/tkinter/ttk.pyi

```diff
@@ -1071,26 +1083,21 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
     @overload
     def tag_bind(self, tagname: str, *, callback: str) -> None: ...
     @overload
-    def tag_configure(self, tagname: str, option: Literal["text"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["image"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["anchor"]) -> tkinter._Anchor | Literal[""]: ...
-    @overload
     def tag_configure(self, tagname: str, option: Literal["foreground", "background"]) -> tkinter._Color: ...
     @overload
-    def tag_configure(self, tagname: str, option: str) -> Any: ...
+    def tag_configure(self, tagname: str, option: Literal["font"]) -> _FontDescription: ...
```

Comment:
Not all possible `_FontDescription` values can be returned:

```python
>>> import tkinter.font
>>> from tkinter.ttk import Treeview
>>> tv = Treeview()
>>> tv.tag_configure('foo', 'font')
''
>>> tv.tag_configure('foo', font=('Helvetica', 12, 'bold'))
{}
>>> type(tv.tag_configure('foo', 'font'))
<class '_tkinter.Tcl_Obj'>
>>> tv.tag_configure('foo', font='TkFixedFont')
{}
>>> type(tv.tag_configure('foo', 'font'))
<class '_tkinter.Tcl_Obj'>
>>> tv.tag_configure('foo', font=tkinter.font.Font())
{}
>>> type(tv.tag_configure('foo', 'font'))
<class '_tkinter.Tcl_Obj'>
```

As you can see, the return value seems to always return `_tkinter.Tcl_Obj` or an empty string. We can change it to describe that if you want, like this:

```suggestion
    def tag_configure(self, tagname: str, option: Literal["font"]) -> _tkinter.Tcl_Obj | Literal[""]: ...
```

But it really doesn't matter in practice, because you will need to either `str()` the return value or pass it to something that expects a `_FontDescription`, and those will work either way.

Reply:
It sounds like `_FontDescription` needs to be updated to include `_tkinter.Tcl_Obj` as an alternative. Even though not every `_tkinter.Tcl_Obj` is a valid `_FontDescription`, the same could be said of `tkinter._TkinterSequence[Any]` which is already in the `_FontDescription` definition. I'm happy to make this change if people think it would be a good idea.

Regardless of how `_FontDescription` is defined, the return value of `tag_configure("foo", "font")` should be suitable to pass to anything else that wants a `_FontDescription`, so I think it makes sense to label the return type as `_FontDescription`.

Reply:
Adding `Tcl_Obj` to `_FontDescription` sounds good. The return type can be `_FontDescription`, so that we can be sure passing it to something expecting a `_FontDescription` works.